content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
+```
+2) Take Jenkins instance backup as a recovery option ::
+[source,bash]
+```
+#Take backup of Jenkins home directory. In my case my Jenkins home directory is /var/lib/jenkins.
+
+cd /var/lib
+tar -cvzf jenkins_date.tar.gz jenkins/
+mv jenkins_date.tar.gz $HOME
+```
+
+3) Take The backup of the current Jenkins war file ::
+[source,bash]
+```
+# Take the backup of Jenkins current version binary using following commands:
+
+cd /usr/share/java/
+cp -p jenkins.war jenkins.war.old
+```
+
+4) Download The New LTS Jenkins Version ::
+[source,bash]
+```
+#Download the new version of Jenkins WAR file using the following commands:
+
+wget https://updates.jenkins-ci.org/latest/jenkins.war
+```
```

Comment:
We **must not upgrade by copying in a new war file** if the previous installation was done with a `deb` or an `rpm`.  Upgrade must be performed by the same type of command as the initial install.  That is specific to the Linux distribution.
```suggestion
```